include/SDL3/SDL_gamepad.h

```diff
@@ -122,6 +122,7 @@ typedef enum SDL_GamepadType
     SDL_GAMEPAD_TYPE_NINTENDO_SWITCH_JOYCON_RIGHT,
     SDL_GAMEPAD_TYPE_NINTENDO_SWITCH_JOYCON_PAIR,
     SDL_GAMEPAD_TYPE_GAMECUBE,
+    SDL_GAMEPAD_TYPE_PSMOVE,
```

Comment:
Let's not add to this enumeration yet. Typically support is added and then if it becomes valuable for a popular controller type, we add to this enumeration later.